thingiverse/ThingiverseService.py

```diff
@@ -123,15 +161,15 @@ def isQuerying(self) -> bool:
         return self._is_querying
 
     @pyqtProperty("QVariant", notify=activeThingChanged)
-    def activeThing(self) -> Dict[str, any]:
+    def activeThing(self) -> Thing:
```

Comment:
This type actually didn't change, as the returned data is a dict representation of the object (otherwise Qt craps out).

Reply:
You can get a dict up & running with "QVariantMap"